examples/nextjs-with-typescript/pages/_document.tsx

```diff
@@ -11,11 +11,13 @@ export default class MyDocument extends Document {
         <Head>
           {/* PWA primary color */}
           <meta name="theme-color" content={theme.palette.primary.main} />
+          <link rel="shortcut icon" href="/static/favicon.ico" />
           <link
             rel="stylesheet"
             href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
           />
-          <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+          {/* Inject MUI styles first to match with the prepend: true configuration. */}
+          {(this.props as any).emotionStyleTags}
```

Comment:
This fixes the initial HTML output (before JS hydration). Improve support of CSS modules and Tailwind CSS

Reply:
Yep, it should fix the issue in #29648. Thanks!